ui/pages/institutional/confirm-connect-custodian-modal/confirm-connect-custodian-modal.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import configureMockStore from 'redux-mock-store';
+import { renderWithProvider } from '../../../../test/lib/render-helpers';
+import ConfirmAddCustodianToken from '.';
+
+describe('Confirm Add Custodian Token', () => {
+  global.platform = { openTab: jest.fn() };
+
+  const mockStore = {
+    metamask: {
+      providerConfig: {
+        type: 'test',
+      },
+      preferences: {
+        useNativeCurrencyAsPrimaryCurrency: true,
+      },
+    },
+    history: {
+      push: '/',
+      mostRecentOverviewPage: '/',
+    },
+  };
+
+  const store = configureMockStore()(mockStore);
+
+  it('shows the modal with its text', () => {
+    renderWithProvider(
+      <ConfirmAddCustodianToken
+        onModalClose={() => console.log('Close')}
+        custodianName="Qredo"
+        custodianURL="https://qredo.com"
+      />,
+      store,
+    );
+
+    const tokenContainer = screen.getByText(
+      "To connect your accounts log into your Qredo account and click on the 'connect to MMI' button.",
+    );
+    expect(tokenContainer).toBeInTheDocument();
```

Comment:
Optional: you can find by testId.So if the text changes in the future, you don't need to modify this test ~

Reply:
Yap, but in this case we prefer to really check the text to make sure it's not changed in the future and has the correct custodian name, in this case Qredo.